src/pages/passwordUpdated.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import TwoThirdsLayout from '../layout/Layout';
+import { deleteCookieOnServerSide } from '../utils';
+import { USER_COOKIE } from '../constants';
+
+const title = 'Password Updated - Fares Data Build Tool';
+const description = 'Password Updated page of the Fares Data Build Tool';
+
+interface PasswordUpdatedProps {
+    redirectTo: string;
+}
+
+const PasswordUpdated = ({ redirectTo }: PasswordUpdatedProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description}>
+        <h1 className="govuk-heading-l">Password Updated</h1>
+        <p className="govuk-body-l">Your password has been updated successfully</p>
+        <a
+            href={redirectTo}
```

Comment:
This should redirect to the My account page

Reply:
You're right - I did see this on the ticket. However, having the /paswordUpdated page route back to the /account page is a bit strange because of the way the /account page is currently designed. Once on the /account page it isn't obvious to the user how to return to the homepage (the only option is to click on the page header). Hence, I thought it better to redirect the user back to the start page rather than /account. Think this is a good thing to discuss as a team and make a decision on how we route the user here.

Reply:
The current flow makes sense which is that it goes back to the start page. I would go with this and see what the other's say.